spec/csv2avro/converter_spec.rb

```diff
@@ -351,15 +351,15 @@
         CSV2Avro::Converter.new(reader, avro_writer, bad_rows_writer, error_writer, { delimiter: "\t" }, schema: schema).convert
       end
 
-      it 'should have the bad data in the original form' do
+      it 'should report the bad rows correctly' do
         expect(bad_rows_writer.string).to eq(
-          "id\ttitle\tdescription\n1\t\tdresses\n4\t\tfemale-shoes\n"
+          "row 2: Missing value at name\nrow 5: Missing value at name\n"
```

Comment:
I would omit the word `row` or replace it with `L2:` which is a standard way of saying "line 2". What's the most readable for whoever looks at this file? I also think a file with the syntax:

```
L2: Missing value at name
```
could be used by editors to navigate to the errors.

What does JSLint and co. output?